basilisk/gui/conversation_tab.py

```
@@ -51,6 +52,7 @@
 
 from .base_conversation import BaseConversation
 from .history_msg_text_ctrl import HistoryMsgTextCtrl
+from .ocr_handler import OCRHandler
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)